### DISCUSSION TITLE
Win 11: smplayer 24.5, how do I tell it to use my 7950x AMD igpu, and not my Nvidia 4090?

### DISCUSSION THREAD

**Discussion by fancellu** [smplayer-dev/smplayer#1065] [Q&A]
Just this. I'd like it to just use my igpu and not the 4090. I have no clue, have tried many things, but task manager always shows it using the 4090

Thanks

**Comment by bendover22** [comment]
Why use only igpu for smp, but the Nvidia card for other graphics?
You may have to disable / remove the Nvidia, for OS to select the igpu.
AFAIK, mostly, if OS sees an installed GPU (& finds drivers), it assumes you want to use it.

Not sure about Win, but in 'Nix, trying to use i-graphics & a GPU card in same session is...???
Less likely to work in Win (not a "flexible" OS).
